rebar.config

```diff
@@ -3,7 +3,6 @@
 	{jiffy, {git, "https://github.com/ArweaveTeam/jiffy.git", {ref, "f25a120f02951b9fd618d1f8f6343fdb8dbe133f"}}},
 	{gun, "1.3.3"},
 	{cowboy, "2.10.0"},
-	{graphql, {git, "https://github.com/shopgun/graphql-erlang.git", {branch, "master"}}},
```

Comment:
I thought we removed this in a previous commit - maybe we need to rebase this branch? (weird that github doesn't seem to think so)